src/meta/src/stream/fragmenter.rs

```diff
@@ -89,8 +91,12 @@ where
         stream_node: &StreamNode,
         ctx: &mut CreateMaterializedViewContext,
     ) -> Result<BTreeMap<FragmentId, Fragment>> {
+        let stream_node = stream_node.clone();
+
         // Generate fragment graph and seal
         self.generate_fragment_graph(stream_node)?;
+        // The stream node might be rewritten after this point. Don't use `stream_node` anymore.
```

Comment:
Can we simply take its ownership in `generate_graph`? Does the caller still needs the origin plan anymore? 🤔

Reply:
Yes. It won't need. After this step, the plan is now fragments in fragment graph.